utest/resources/Listener.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+
+class Listener:
+    ROBOT_LISTENER_API_VERSION = 2
+
+    def __init__(self, name):
+        self.name = name
+
+    def start_suite(self, name, attrs):
+        sys.__stdout__.write("[from listener {}]\n".format(self.name))
+
+    def close(self):
+        sys.__stdout__.write("[listener close]\n")
```

Comment:
Is calling this `close` method actually tested somewhere? Should it be added to both of the newly added tests?